Modules/Projects/Core/Scene_TownInn.json

```diff
@@ -814,7 +814,7 @@
 										"Tooltip": "Talk to the big fuzzy-looking taur in the corner.",
 										"Highlight": false,
 										"ButtonWidth": 1.0,
-										"ScriptAction": "LogGsub(\"INN_FOXTAUR_APPROACH\", {partner = ftpartner})",
+										"ScriptAction": "Text.SetContext(\"partner\", ftpartner)\r\nLog(\"INN_FOXTAUR_APPROACH\")",
```

Comment:
That's a whole bunch of `LogGsub`s gone. Good riddance, shouldn't be much longer until those old functions can be removed entirely.